portfolio/src/main/webapp/index.html

```diff
@@ -19,102 +19,155 @@ <h1 class="page_headers">Ricardo Chapa's Portfolio</h1>
-            src="/../images/profile_picture.jpg" 
-            alt="Picture taken in the library at Tecnologico de Monterrey of me, 
-                smiling and wearing a hoodie with the RoBorregos logo in it (A robotic Lamb).">
-          </img>
-          <!-- Information (Text) Container -->
-          <div id="paragraph_container">
-            <p class="justify_text">
-              My name is Ricardo Chapa Romero, I'm 19 years old 
-              and I live in Mexico. I'm a STEP intern at Google
-              and a fun fact about me is that my record for sleeping time is 25 hours!
-            </p>
-            <p class="justify_text">
-              I'm a Digital Systems and Robotics student at ITESM, currently in 5th semester.
-              For the past 2 years I've been an active participant at the universities robotics team: RoBorregos.
-              The opportunity of being part of the robotics team has allowed me to learn various topics like Web Development,
-              Machine Learning and Robot Oriented Programming.
-            </p>
-            <p class="justify_text">
-              I really like playing videogames and watching TV series in my free time,
-              also enjoy a good time with my friends and familiy.
-            </p>
-          </div>
-          <hr>
-          <!-- About me container -->
-          <div id="about_me_container">
-            <h2 class="page_headers">Projects I've worked in</h2>
-            <!-- Projects Containers -->
-            <div id="projects_container">
-                <ul>
-                  <li><b>RoBorrego's Warehouse: </b>Worked as PM and Front End developer.</li>
-                  <li><b>RoBorrego's Webpage: </b>Worked as Front End dveloper.</li>
-                  <li><b>English Course for ITESM: </b>Worked as Front End developer.</li>
-                  <li><b>RoboCup @Home Challenge: </b>Worked on the Navigation Team.</li>
-                  <li><b>RoboCup Soccer Lightweight Tournament: </b>Worked on Movements and Sensors of the Robots</li>
-                </ul>
+      <div id="page_wrapper">
+        <div id="comment-wrapper">
+          <!-- Comment Container -->
+          <div id="comments_container">
+            <div id="comment_section_wrapper">
+              <form action="/new-comment" method="POST">
+                <h2 class="page_headers">Leave a comment!</h2>
+                <textarea 
+                  type="text" 
+                  name="user-comment" 
+                  id="comment-section"
+                  placeholder="Add your comment here!"
+                ></textarea>
+                <input 
+                  type="text" 
+                  name="user-name" 
+                  id="comment-section"
+                  placeholder="Add your name (optional)"
+                />
+                <br/>
+                <br/>
+                <input type="submit" alt="Click to submit your comment" id="comment_submit"/>
```

Comment:
Style guide: Do not close void elements, i.e. write `<input>`, not `<input/>`.

Reply:
Thank you for the reference!